e2e/helper.go

```diff
@@ -0,0 +1,123 @@
+package e2e
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"text/template"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	WorkloadIdentityNamespace = "azure-workload-identity-system"
```

Comment:
```suggestion
	AzureWorkloadIdentityNamespace = "azure-workload-identity-system"
```
In general please use `Azure` prefix for all Azure related stuff.

Reply:
Done.